### PR TITLE
XS-4544 Adding extension member to checked axes.

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Added InvestmentsNetAssetValueMember to FairValueByFairValueHierarchyLevelAxis
- Changed travis.yml to install regex

### Pull request process

- [ ] Full description of changes provided above.
- [ ] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/dqc_us_0001.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from arelle.FileSource import saveFile, openFileSource
 
 _CODE_NAME = 'DQC.US.0001'
-_RULE_VERSION = '3.1.0'
+_RULE_VERSION = '3.1.1'
```
Comment (davidtauriello) [inline dqc_us_rules/dqc_us_0001.py]: @allenbross-wf - please remember to increment this to 3.2.0 prior to commit

**Comment by brianmyers-wf** [comment]
just unfamiliar with these tests at this point... what do the json's in the resources directory control? looks like massive changes to them and not comfortable signing off without context around them :\

**Comment by allenbross-wf** [comment]
The json files setup the parameters for each test.  They define, for a given axis, what members can or cannot be used, whether extension are allowable, etc. 
The reason the change is so jarring this time is instead of returning things as they are dumped out of functions they are now alphabetically sorted.

**Comment by brianmyers-wf** [comment]
sounds good, thanks for the explanation.
+1

**Comment by derekgengenbacher-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
QA + 10 - the doc attached in XS-4547 did not fire NN checks any longer.  Merging.

**Pull request merged**